### DISCUSSION TITLE
Query Regarding Project Submission

### DISCUSSION THREAD

**Discussion by AmanJulka** [github/india#713] [Students]
Hello Team,
Can I host my project on Versel?
Thank you.

**Comment by Ashujha2002** [comment]
How to complete a course i am 2024 batch student i have registered this on 15 jan 2022
And i am not able to complete the course 
So what are the possibile ways to complete the course and hiw to make a project @AmanJulka plz reply sir

**Reply by Ashujha2002** [reply]
@future ready talent
